taglib/mpeg/id3v2/id3v2framefactory.h

```diff
@@ -137,7 +137,7 @@ namespace TagLib {
       /*!
        * Destroys the frame factory.
        */
-      virtual ~FrameFactory();
+      ~FrameFactory();
```

Comment:
`FrameFactory` is meant to be subclassed and has virtual methods, so the destructor should be virtual too.